tracing-subscriber/src/fmt/format/mod.rs

```diff
@@ -1239,26 +1239,34 @@ impl<'a> field::Visit for DefaultVisitor<'a> {
             return;
         }
 
-        self.maybe_pad();
         self.result = match field.name() {
-            "message" => write!(self.writer, "{:?}", value),
+            "message" => {
+                self.maybe_pad();
+                write!(self.writer, "{:?}", value)
+            }
             // Skip fields that are actually log metadata that have already been handled
             #[cfg(feature = "tracing-log")]
             name if name.starts_with("log.") => Ok(()),
```

Comment:
Perhaps it would be clearer if we perform the check for `field.name().starts_with("log.")` before the `match` statement, and return `Ok(())` there directly, then we can keep the `self.maybe_pad()` in only one place.

What do you think?

Reply:
Sure. I was trying to leave the behaviour of the code as alone as possible, but I can make that change.

Reply:
@hds this is now done.